### DISCUSSION TITLE
Where can I know version OpenBrush for exact oculus quest 2 os version ?

### DISCUSSION THREAD

**Discussion by mrmoroshkin** [icosa-gallery/open-brush#465] [Q&A]
Where can I know version OpenBrush for exact oculus quest 2 os version ?
I have OpenBrush v 1.0.0 and oculus quest 2 os version 37.

But I interested to upgrade to openbrush 2, download, install via SideQuest - bun run and not worked, start window didn't showup.
Downgrage to 1.0.0.

And, where can I know version OpenBrush for exact oculus quest 2 os version ?
For example OpenBrush v 1.0.0 work from oculus quest 2 os version 37.
For example OpenBrush v 2.2.1 work from oculus quest 2 os version ??? and so on.

Best regards!

**Comment by andybak** [comment]
Sorry this was unanswered for so long. I didn't get a notification and there's not much traffic on our Github Discussions.

Sounds like you want to install the beta version: https://docs.openbrush.app/alternate-and-experimental-builds/open-brush-beta-docs

Does that answer your question?
